### DISCUSSION TITLE
Added: LDM Notes for June 2nd, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4799] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-06-02.md

## Agenda

1. Enhanced #line directives
2. Lambda return type parsing
3. Records with circular references

**Comment by MgSam** [comment]
FWIW, T4 Templates also use `#line` mapping. And yes, VS still supports them :)

**Reply by 333fred** [reply]
I'm certain they do, and hopefully I was clear that the `#line` enhancements aren't just for Razor. They _are_ the user driving the changes, however.
